sdk/modelsrepository/Azure.Iot.ModelsRepository/src/DtmiConventions.cs

```diff
@@ -7,6 +7,10 @@
 
 namespace Azure.Iot.ModelsRepository
 {
+    /// <summary>
+    /// <c>DtmiConventions</c> implements the core aspects of the IoT model repo conventions
```

Comment:
I don't understand the use of `<c>` tags around types. What's the intent here?

When we reference a type that is not the class we're describing, we use `<see ../>` tags. When describing the current one, no tags should be necessary.

Reply:
The intent is to have sharper looking docstrings. 

https://docs.microsoft.com/en-us/dotnet/csharp/codedoc#c  -- "...you use the <c> tag for marking part of text as code"

So it would be slightly elevated in elegance in the same way we can use tildes in github `markdown`

Reply:
Is that a standard we use in our web docs generally? That's why I ask. I don't think I've seen that be the standard.

Reply:
Ok fair point. I'll take out the tags to conform.